paper/paper.md

```diff
@@ -20,21 +20,22 @@ authors:
+With E2EDNA 2.0 our goal is to create an all-python simulation package for single-stranded DNA with small ligands that is easy to install and use in python-based workflows. The pipeline is automated, yet flexible, taking us from DNA aptamer sequence to folded aptamer, and aptamer-ligand complex. Currently, available softwares designed for computationally studying DNA aptamers predominantly focus on partial feature analyses of RNA and DNA aptamers. For example, APTANI [@Caroli:2016] and APTANI2 [@Caroli:2020], commonly used by the aptamer community, select potentially relevant aptamers from SELEX (Systematic Evolution of Ligands by EXponential enrichment)[@Tuerk:1990; @McKeague:2014] experimental data sets through a sequence-structure analysis, and AEGIS, a platform equipped with a generative deep learning model to propose novel aptamer sequences [@Biondi:2018]. These approaches are aimed at fast black-box analysis of large numbers of sequences. A detailed analysis of a small number of high-promise candidate sequences is often required but no automated and easy-to-use simulation package exists in the computational space aside from E2EDNA [@Kilgour:2021]. 
 
-The gap that E2EDNA family of programs addresses is the absence of a non-black-box one-stop-shop aptamer simulation packages, which are capable of providing in silico predictions of 2D structure, 3D structure, and aptamer-ligand binding simulation while keeping methodology flexible and analysis open-ended. E2EDNA 2.0 achieves this predominantly using python which makes installation and access easier for users than the original E2EDNA package.
+The gap that E2EDNA family of programs addresses is the absence of a non-black-box one-stop-shop aptamer simulation packages, which are capable of providing in silico predictions of 2D structure, 3D structure, and aptamer-ligand binding simulation while keeping methodology flexible and analysis open-ended. E2EDNA 2.0 achieves this in two key ways: an all-python implementation which makes installation and access easier for users than the original E2EDNA package, and the automation of the dozens of tasks required in setting up, running, and interpreting atomistic aptamer-ligand simulations.
```

Comment:
We do not perform post-simulation analysis on aptamer-ligand trajectory. Therefore, perhaps rephrase or even remove "interpreting atomistic aptamer-ligand simulations"?

Reply:
I wondered about that indeed as I was putting it in. I do want to emphasize that the tools are available, though as you point out the automated analysis is omitted. maybe "preparing the analysis" or "setting up the analysis"? Or is that too weasel-y

Reply:
I would add this after 'original E2EDNA package': which required interface with Tinker9(?).

Reply:
Michael's statement is that it's making automation of all these good things easiers. It does even if indirectly so I think here this works ok

Reply:
Okay, sounds good. We are talking about the capability of the pipeline - it covers "trajectory interpretation".